### DISCUSSION TITLE
The packages `next` and `create-next-app` have dissappeared

### DISCUSSION THREAD

**Discussion by zemd** [vercel/next.js#73661] [Help] [answered]
### Summary

if you run `npx create-next-app@latest test1` or navigate to https://www.npmjs.com/package/next and https://www.npmjs.com/package/create-next-app you get an 404 error:

```
npm error code E404
npm error 404 Not Found - GET https://registry.npmjs.org/create-next-app - Not found
npm error 404
npm error 404  'create-next-app@latest' is not in this registry.
npm error 404
npm error 404 Note that you can also install from a
npm error 404 tarball, folder, http url, or git url.
```

**Comment by icyJoseph** [comment, marked as answer]
They are back online! Or perhaps never were offline? NPM is undergoing maintenance: 

```
Dec 8, 2024
Unresolved incident: Scheduled infrastructure upgrade.
```

<img width="975" alt="Screenshot 2024-12-08 at 22 17 21" src="https://github.com/user-attachments/assets/1f8b541d-61ef-424f-bd59-234adfdc8edd">

**Reply by zemd** [reply]
thanks
